.github/feature.md

```diff
@@ -1,3 +1,13 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
We should undo this file's changes. I'll try to look into a way to exclude files from our ToC generation today

Reply:
In the meantime we'll have to make note